collections/Add_A/MS_Add_A_263.xml

```diff
@@ -37,13 +41,39 @@
+                     <supportDesc>
+                        <extent> iii + 48 + 16 + i leaves
+                           <dimensions type="leaf" unit="mm">
+                              <height min="240" max="245">240–5</height>
+                              <width max="175" min="170">170–5</width>
+                           </dimensions></extent>
+                        <foliation> 1–64 in late 19th-century purple pencil, i–iii and 65v in 20th-century pencil</foliation>
+                     </supportDesc>
+                  </objectDesc>
+                  <bindingDesc>
+                     <binding contemporary="true">
+                        <p>Contemporary binding. Sewn on four pairs of twisted parchment bands, laced and pegged separately into horizontal channels in wood (beech?) boards, covered with undecorated tawed(?) skin, overlapping the edges of the leaves; two strap-and-pin clasps (closing from the back to the front board); the front cover inscribed with a shelfmark in bold 15th-century characters ‘Y.22.’; the top of the spine inscribed ‘Legen<lb/>da S. | Thom<lb/>as [ … ] | &amp; | S. Aug. | S. Ambr.’ (worn; readings uncertain; cf. fols. 49–54).
+                        </p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
+               <history>
+                  <provenance>Doubtless written for a Dominican house in Germany, and perhaps still there in the 15th century.</provenance>
+                  <provenance>Annotated in German, 14th(?)-century: <q>in götlicher veruck<gap reason="illegible" unit="char" quantity="2"></gap></q> (fol. 24v, lower margin).</provenance>
+                  <provenance>Unidentified library, with its 15th century shelfmark <q type="pressmark">Y .22.</q> inscribed on the front cover and fol. 1r, the latter altered from <q type="pressmark">M .12.</q>.</provenance>
+                  <provenance>Unidentified 19th(?)-century collection, doubtless German: inscribed in ink <q type="pressmark">N XXIV</q> and <q type="pressmark">N 24</q> (fol. ii r).</provenance>
+                  <provenance>Unidentified 19th-century German-speaking owner/bookseller: with an inserted slip inscribed in pencil ‘Beide Theile dieser Codex 14 saec.’ (attached to fol. ii r); his(?) number ‘173’ inside the back board and on a slip of paper attached to the back board.</provenance>
+                  <provenance>H. Grevel &amp; Co., London bookseller: their(?) inventory number ‘9157’ in pencil (fol. iir upper right; fol. 1r, top left; fol. 64r, lower left); their Catalogue 4, <title>Catalogue of rare manuscripts and early printed books …</title> (1884), item 6; pencil price ‘£4’ (fol. iir):</provenance>
+                  <acquisition when="1884">Purchased by the Bodleian on 24 March 1884, for £3 10s; inscribed in pencil ‘Grevel 1884’ (fol. iir). Former Bodleian shelfmarks: ‘MS. Bodl. Adds. A. 263’ (fol. 1r, lower margin; cancelled by encircling).
```

Comment:
Noting 'Adds. A. 263' as a former shelfmark seems pedantic to me, but is there significance that I am missing?

Reply:
agreed but see comment under Canon Pat Lat 113